com.unity.render-pipelines.high-definition/Documentation~/UI-Best-Practices.md

```diff
@@ -1,14 +1,20 @@
 # Working with UI in the High Definition Render Pipeline
 
-Almost every Project uses some form of user interface (UI). This could be: in a menu system, to indicate a point of interest in your application, or to display valuable information to users. This guide explains how to use [Unity UI](https://docs.unity3d.com/Packages/com.unity.ugui@latest) in your High Definition Render Pipeline (HDRP) Project as well as provide best practice information to keep in mind when working with it.
+Almost every Project uses some form of user interface (UI). This could be: 
+
+* In a menu system
+* To specify a point of interest in your application
+* To display valuable information to users. 
+
+This guide explains how to use [Unity UI](https://docs.unity3d.com/Packages/com.unity.ugui@latest) in your High Definition Render Pipeline (HDRP) Project and provide best practice information to keep in mind when working with it.
 
 HDRP supports all the Canvas **Render Modes**, however, it only supports Unlit UI shaders for Canvas UI.
 
 ## Working in a linear color space
 HDRP Projects require you to work in linear [color space](https://docs.unity3d.com/Manual/LinearLighting.html). When you create UI in the Editor, ensure the sprite textures are in linear color space to make sure the UI renders correctly. This is especially important for transparent sprites. For more information, see [Linear or gamma workflow](https://docs.unity3d.com/Manual/LinearRendering-LinearOrGammaWorkflow.html) and [Working with linear Textures](https://docs.unity3d.com/Manual/LinearRendering-LinearTextures.html).
 
 ## Multiple camera setup
-HDRP only supports a single camera setup by default. If you need to use more than one camera at the same time, it is best practice to use one the following alternatives:
+HDRP only supports a single camera setup by default. If you need to use more than one camera at the same time, you should use one the following alternatives:
```

Comment:
Should Camera be capitalised here? Also remove "you should" entirely.

Reply:
In the style guide: "Do not use the word should because it is ambiguous. Be explicit. "